lti_consumer/migrations/0011_courseeditltifieldsenabledflag.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.20 on 2021-05-05 11:56
+import logging
+
+import django.db.models.deletion
+import opaque_keys.edx.django.models
+from django.conf import settings
+from django.db import migrations, models
+
+log = logging.getLogger(__name__)
+
+
+class CreateModelIfNotExists(migrations.CreateModel):
+    """
+    Creates the database table if it doesn't already exist.
+
+    This can be used to move a database model from one app to another.
+    """
+
+    def database_forwards(self, app_label, schema_editor, from_state, to_state):
+        # Get the name of the database table
+        db_table = to_state.apps.get_model(app_label, self.name)._meta.db_table
+        # If the database table for this model already exists, do nothing, otherwise
+        # create the table as usual.
+        if db_table in schema_editor.connection.introspection.table_names():
+            log.info(f"{db_table} already exists. Skipping creation.")
+        else:
+            super().database_forwards(app_label, schema_editor, from_state, to_state)
+
+    def database_backwards(self, app_label, schema_editor, from_state, to_state):
+        # Do nothing when applying this is reverse because the original creation of this
+        # table was handled by edx-platform, so we don't want to delete the table on reversal.
+        pass
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('lti_consumer', '0010_backfill-empty-string-lti-config'),
+    ]
```

Comment:
I didn't want to add a dependency here because that might break testing this in environments without access to edx-platform. 

It shouldn't matter what order they are run in since there are two possibilities:

1. The table already exists. In this case, the edx-platform migration have already run, and we will skip creating the table here. 
2. The table doesn't exist. In this case, the table will be created here.